spec/advanced_search_spec.rb

```diff
@@ -402,8 +402,14 @@
       end
       it 'add topic feature films' do
         resp = solr_resp_doc_ids_only({ 'fq' => 'format:("Video"), language:("English"), building_facet:("Green"), topic_facet:("Feature films")', 'q' => 'collection:*' }.merge(solr_args))
-        expect(resp.size).to be >= 25
-        expect(resp.size).to be <= 100
+        # 655a was taken out of topic facet, hence smaller number of records expected.
```

Comment:
this comment will become outdated quickly.  It's helpful to me ... but perhaps it should be a comment here in the PR, rather than in the code?  What do you think?